### PR TITLE
feat(perf-issues): Add n_plus_one to FieldKey.ISSUE_TYPE values

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/stores/tagStore.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ const storeConfig: TagStoreDefinition = {
       [FieldKey.ISSUE_TYPE]: {
         key: FieldKey.ISSUE_TYPE,
         name: 'Issue Type',
-        values: [],
+        values: ['n_plus_one'],
```
Comment (gggritso) [inline static/app/stores/tagStore.tsx:87]: Did you mean
```suggestion
        values: ['n_plus_one_db'],
```
? That's the one we're detecting right now

Comment (evanpurkhiser) [reply static/app/stores/tagStore.tsx:87]: hmmmmmmmmm

Comment (gggritso) [reply static/app/stores/tagStore.tsx]: It actually might be _both_ things, 🤔 `performance_n_plus_one_db`

Comment (evanpurkhiser) [reply static/app/stores/tagStore.tsx]: `performance_n_plus_one_db_queries` is what it actually is 🤔 🤔 🤔 🤔 🤔

**Review by gggritso — APPROVED** [review]
👍🏻

**Pull request merged**